src/utils.jl

```diff
@@ -228,20 +228,45 @@ macro cplwarn(code, message)
     end
 end
 
+function showprogress(progress, message = "")
+    print("$(round(Int, progress)*100)..$message")
+    return true
+end
+
+function dummyprogress(progress, message = "")
+    return true
+end
+
+function progressfunc_wrapper(
+    dfComplete::Cdouble,
+    pszMessage::Cstring,
+    pProgressArg::Ptr{Cvoid},
+)::Cint
+    pProgressArg == C_NULL && return true
+    f = unsafe_pointer_to_objref(pProgressArg)
+    isa(f, Ref) || return true
```

Comment:
Can you provide a comment above this line explaining it?

Reply:
Resolved by https://github.com/yeesian/ArchGDAL.jl/pull/353/commits/24c1304555c48b8e4931b9755e3113f35e312a52